tox.ini

```diff
@@ -40,7 +40,7 @@ basepython =
     python2.7
```

Comment:
I would suggest this should be 3.6, to match .travis.yml

Reply:
Fixed.